recipes/icc/all/test_package/example.cpp

```diff
@@ -0,0 +1,7 @@
+#include <icc/Component.hpp>
+#include <icc/Event.hpp>
+#include <icc/Context.hpp>
+
+int main() {
+  return 0;
```

Comment:
Please include something to test the linkage

https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#minimalistic-source-code